release.yml

```diff
@@ -0,0 +1,59 @@
+pool:
+  vmImage: 'windows-latest'
+
+variables:
+  solution: '**/*.sln'
+  testProjects: "**/*Tests/*.csproj"
+  publishProject: "**/*Tool/*.csproj"
+  buildConfiguration: 'Release'
+
+steps:
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet build'
+  inputs:
+    command: build
+    projects: '$(solution)'
+    arguments: '--configuration $(buildConfiguration)'
+
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet test'
+  inputs:
+    command: test
+    projects: '$(testProjects)'
+    arguments: '--configuration $(buildConfiguration) --collect "Code coverage"'
+    
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet publish'
+  inputs:
+    command: 'publish'
+    projects: '$(publishProject)'
+    arguments: '--configuration $(buildConfiguration) --output $(Build.ArtifactStagingDirectory)'
+    publishWebProjects: false
+    zipAfterPublish: True
+
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet pack nugets'
+  inputs:
+    command: pack
+    configuration: '$(buildConfiguration)'
+    packDirectory: '$(build.artifactStagingDirectory)/nupkgs'
+    versioningScheme: 'byEnvVar'
+    versionEnvVar: 'nuget_version'
+    nobuild: true
+    zipAfterPublish: true
+  env:
+    nuget_version: 2.0.0-$(build.buildNumber)
```

Comment:
Can we extract version number from Directory.Build.props? Seems for Anonymizer.core nuget, we don't need buildNumber in version tag as release only happens after every version bump?

Reply:
Thanks, updated the pipeline to get version from Directory.Build.props. For the build number, I would prefer to have a minor version which would increase by build. It would help always help us separate different builds

Reply:
Fine. Got it.